README.md

```diff
@@ -43,33 +43,23 @@ A help message is displayed when passing no arguments, `-h`, or `--help`.
 ```
 user@hostname ~> chwifi
 Usage: chwifi [OPTION]... <PROFILE>
-Connect to home or work wireless networks, caching rolling passwords at work
+Connect any network-manager wireless networks, caching rolling passwords at work
```

Comment:
```suggestion
Connect to any netctl wireless networks, caching rolling passwords at work
```

You forgot 'to' and let's keep the help-message specific until broader support is implemented.